setup_docker.sh

```diff
@@ -1,5 +1,9 @@
 #kills the running process, and creates a new web-server to run
 docker kill "$(docker ps -q)"
+docker stop $(docker ps -a -q)
+docker rm $(docker ps -a -q)
+docker rmi $(docker images -a -q)
 docker load -i webserver_img
-docker run -d -p 80:2020 httpserver ./web-server simple_config
+docker run -d -p 80:2020 httpserver:latest ./web-server simple_config
```

Comment:
80 is the port that our aws machine uses. just for ease of use of creating http requests to the ec2 instance (don't have to specify port, because port 80 is default).